extensions/Particle3D/PU/CCPUParticleSystem3D.h

```diff
@@ -179,22 +196,17 @@ struct CC_DLL PUParticle3D : public Particle3D
     }
     bool freezed;
 
-    // Time to live, number of seconds left of particles natural life
+    ///< Time to live, number of seconds left of particles natural life
     float timeToLive;
 
-    // Total Time to live, number of seconds of particles natural life
+    ///< Total Time to live, number of seconds of particles natural life
     float totalTimeToLive;
 
-    // The timeFraction is calculated every update. It is used in other observers, affectors, etc. so it is
-    // better to calculate it once at the Particle level.
+    ///< The timeFraction is calculated every update. It is used in other observers, affectors, etc. so it is
+    ///< better to calculate it once at the Particle level.
     float timeFraction;
 
-        /*  Mass of a particle.
-    @remarks
-        In case of simulations where mass of a particle is needed (i.e. exploding particles of different
-        mass) this attribute can be used.
-    */
-    float mass;
+    float mass;///< mass of a particle
```

Comment:
The remarks can help us understand the property well. We would better try to keep them. Is there @remarks in the doxygen? You can treat it as comment of variables if not.